en/views/json-and-xml-views.rst

```diff
@@ -162,7 +162,54 @@ Creating JSON Views
 The JsonView class supports the ``_jsonOptions`` variable that allows you to
 customize the bit-mask used to generate JSON. See the
 `json_encode <http://php.net/json_encode>`_ documentation for the valid
-values of this option.
+values of this option.::
+
+    // Controller code
+    class VideosController extends AppController
+    {
+    
+        //....
+        
+        public function export($acction = '')
+        {
+            $acction = mb_convert_case($acction, MB_CASE_LOWER);
+            
+            // Accion Out ..?
+            switch ($acction) {
+            
+                case 'json':
+                case 'xml':
+                    // Out file JSON ..?
+                    if($acction === 'json'){
+                        // Out json
+                        $this->viewBuilder()->className('Json');
+                    }else{
+                        // Out xml
+                        $this->viewBuilder()->className('Xml');
+                    }
```

Comment:
OK, I use "CASE" instead of "IF" because it is part of a code mine that has other formats that are not native to CakePHP with expert libraries.
I'll change it to "IF"
Greetings and thanks